core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -711,13 +718,40 @@ public void lostFilesDetectionTest() throws Exception {
 
     // run the detector
     HeartbeatScheduler.schedule(HeartbeatContext.MASTER_LOST_FILES_DETECTION);
-    Assert.assertTrue(HeartbeatScheduler.await(HeartbeatContext.MASTER_LOST_FILES_DETECTION, 5,
-        TimeUnit.SECONDS));
+    Assert.assertTrue(HeartbeatScheduler
+        .await(HeartbeatContext.MASTER_LOST_FILES_DETECTION, 5, TimeUnit.SECONDS));
 
     fileInfo = mFileSystemMaster.getFileInfo(fileId);
     Assert.assertEquals(PersistenceState.LOST.name(), fileInfo.getPersistenceState());
   }
 
+  /**
+   * Tests load metadata logic.
+   *
+   * @throws Exception if a {@link FileSystemMaster} operation fails
+   */
+  @Test
+  public void testLoadMetadataTest() throws Exception {
+    FileUtils.createDir(Paths.get(mUnderFS).resolve("a").toString());
+    mFileSystemMaster.loadMetadata(new AlluxioURI("alluxio:/a"),
+        LoadMetadataOptions.defaults().setCreateAncestors(true));
+    mFileSystemMaster.loadMetadata(new AlluxioURI("alluxio:/a"),
+        LoadMetadataOptions.defaults().setCreateAncestors(true));
+
+    // TODO(peis): Avoid this hack by adding an option in getFileInfo to skip loading metadata.
+    mThrown.expect(FileAlreadyExistsException.class);
+    mFileSystemMaster
```

Comment:
@peisun1115 this line will throw an exception, so all code after it is dead. Please add documentation about what this test is doing and fix the issue with the dead code (probably by using try-catch instead)